packages/paste-core/primitives/listbox/src/ListboxPrimitiveItem.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import type {CompositeItemProps} from '@twilio-paste/reakit-library';
+import {CompositeItem} from '@twilio-paste/reakit-library';
+
+import type {As} from './types';
+
+export interface ListboxPrimitiveItemProps
+  extends Omit<CompositeItemProps, 'unstable_virtual' | 'unstable_moves' | 'unstable_system' | 'wrapElement' | 'wrap'> {
+  selected?: boolean;
+  /** Event handler to respond to selection events */
+  onSelect?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  as?: As;
+}
+
+export const ListboxPrimitiveItem = React.forwardRef<HTMLButtonElement, ListboxPrimitiveItemProps>(
+  ({selected = false, onSelect, ...props}, ref) => (
+    <CompositeItem as="button" {...props} ref={ref} role="option" aria-selected={selected} onClick={onSelect} />
```

Comment:
thought: should we add `type="button"` into here after the `{...props}` to prevent people making this `type="submit"` by accident?

Reply:
Sure, wouldn't hurt!